charts/generic-service/values.yaml

```diff
@@ -43,14 +43,13 @@ service:
 
 ingress:
   enabled: false
+  v0_47_enabled: true
   annotations:
     nginx.ingress.kubernetes.io/custom-http-errors: "418"
     external-dns.alpha.kubernetes.io/aws-weight: "100"
   host: chart-example.local
   path: /
   tlsSecretName: chart-example-tls
-  # live-1 is blue, live is green
-  contextColour: blue
```

Comment:
taken out the colour as it is now always green (blue was for the old live-1 namespaces that don't exist anymore)